android/src/main/java/com/thebluealliance/androidclient/Utilities.java

```diff
@@ -1,31 +1,36 @@
 package com.thebluealliance.androidclient;
 
+import com.google.common.base.Predicate;
+
+import com.thebluealliance.androidclient.activities.GamedayActivity;
```

Comment:
You can import the project style file: https://github.com/the-blue-alliance/the-blue-alliance-android#code-style